### PR TITLE
Testing To Do list: Part 1

### PR DESCRIPTION
**Pull request by christianonoh**
- Create a test file
- Added a test for add and remove items
- Installed babelrc for import and export function
- Updated package.json for babel

### CODE REVIEW

**Review by zainsadaqat — COMMENTED** [review]
# Invalid Code Review Request ❌ ❌ ❌
# Hello @christianonoh 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Something went wrong 

- ### You have submitted a project in which all linters are not passing the test. Kindly make all linters look Green! 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: test/addDeleteTask.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+import addTask from '../src/modules/addTask.js';
+import deleteTask from '../src/modules/deleteTask.js';
+import {
+  displayTasks, updateLocalStorage, tasksArray, reindexTasks, displayAllTasks,
+} from '../src/modules/task.js';
+
+jest.mock('../__mock__/testlist.js');
```
Comment (DammyShittu) [inline test/addDeleteTask.test.js:10]: - [ ] Your tests do not run. Kindly rectify that.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @christianonoh,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @christianonoh, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**